common/reviews/api/rush-azure-storage-build-cache-plugin.api.md

```diff
@@ -35,7 +35,9 @@ export abstract class AzureAuthenticationBase {
     // (undocumented)
     deleteCachedCredentialsAsync(terminal: ITerminal): Promise<void>;
     // (undocumented)
```

Comment:
updated types here to allow for a subset of LoginFlowType keys to be defined in the Record.
else, bumping rush version would break type checks if only some of the LoginFlowTypes were defined in the record.